libr/parse/filter.c

```diff
@@ -401,6 +413,20 @@ static bool filter(RParse *p, ut64 addr, RFlag *f, RAnalHint *hint, char *data,
 				}
 				break;
 			}
+			if(bits==16&&x86&&*pnum==':'){
```

Comment:
space between operators and operands

```suggestion
                        if (x86 && bits == 16 && *pnum == ':') {
```